java/integTest/src/test/java/glide/SharedCommandTests.java

```diff
@@ -247,4 +251,21 @@ public void set_missing_value_and_returnOldValue_is_null(BaseClient client) {
         String data = client.set("another", ANOTHER_VALUE, options).get();
         assertNull(data);
     }
+
+    @SneakyThrows
+    @ParameterizedTest
+    @MethodSource("getClients")
+    public void mset_mget_existing_non_existing_key(BaseClient client) {
+        String key1 = RandomStringUtils.randomAlphabetic(10);
+        String key2 = RandomStringUtils.randomAlphabetic(10);
+        String key3 = RandomStringUtils.randomAlphabetic(10);
+        String nonExisting = RandomStringUtils.randomAlphabetic(10);
+        String value = RandomStringUtils.randomAlphabetic(10);
+        Map<String, String> keyValueMap = Map.of(key1, value, key2, value, key3, value);
+
+        assertEquals(OK, client.mset(keyValueMap).get(10, SECONDS));
+        assertArrayEquals(
+                new String[] {value, value, null, value},
+                client.mget(new String[] {key1, key2, nonExisting, key3}).get(10, SECONDS));
```

Comment:
```suggestion
                client.mget(new String[] {key1, key2, nonExisting, key3}).get());
```
`@Timeout(10)` is specified in the file which will timeout any test that last longer than 10 seconds.